camviewer_ui_impl.py

```diff
@@ -2602,6 +2613,8 @@ def getConfig(self):
                 self.cfg.add("markers", self.options.marker)
             if self.options.camcfg is not None:
                 self.cfg.add("dispspec", self.options.camcfg)
+            # if self.options.displayFormat is not None:
```

Comment:
Now, self.options is the command line parameters that we have previously parsed.  It's not unreasonable to say you could pass in the displayFormat on the command line... but if you are going to do this, you need to change the option parser setup to include "displayFormat" as an acceptable option that takes a string argument.  Either do that, or remove this.  (I'd remove this... I think displayFormat is sufficiently obscure that it doesn't need to be a command line argument, but the choice is yours, and I'd approve either way.)

Reply:
I think i will opt to just remove this